src/Base/MultiFab.H

```diff
@@ -493,6 +493,31 @@ factory :
         ;
     }
 
+    py_MultiFab
+        .def("copymf",
```

Comment:
for some reason, calling this "copy" did not seem to work 
May be there is another copy method that is shadowing this 
so I renamed it as copymf and that works!

Reply:
Is this different than the copy method exposed [here](https://github.com/AMReX-Codes/pyamrex/blob/development/src/Base/MultiFab.cpp#L106)?

Reply:
What if you use the extension method [here](https://github.com/AMReX-Codes/pyamrex/blob/development/src/amrex/extensions/MultiFab.py#L654), as in [this](https://github.com/AMReX-Codes/pyamrex/blob/development/tests/test_multifab.py#L492) test?